oat-sa/extension-tao-mediamanager
# fix: support old inline figure

Related to: https://oat-sa.atlassian.net/browse/AUT-2673

Depends on:

- [ ] https://github.com/oat-sa/tao-item-runner-qti-fe/pull/341

Issue with edit old inline figure. On customer env serial doesn't contain `figure_`. And figure element contains `bdy.elements`
![image](https://user-images.githubusercontent.com/25976342/205657657-e4460e79-aed3-4eb2-a12c-61b23ace1083.png)

**How to test:**
1. set tao v50.24.6, taoQtiItem v29.14.2, taoMediaManager v12.30.0 (run `npm i` in views)
2. create passages with inline images in A-Block (it will be block element, saved as figure), save
3. update tao, taoQtiItem, taoMediaManager to develop (run `npm i` in views)
4. update `taoMediaManager/views/js/qtiCreator/renderers/Img.js`and `taoQtiItem/views/js/qtiCreator/renderers/Img.js` remove part `|| $container.parent('span').length && $container.parent('span').data('serial') && $container.parent('span').data('serial').includes('figure')` because on customer env serial doesn't contain `figure_`
5. open passages in Authoring, see errors
![image](https://user-images.githubusercontent.com/25976342/205861804-2e55124e-ca84-4e0b-8b3f-f7d59923bb2c.png)
6. update taoQtiItem, taoMediaManager to `fix/AUT-2673/support-old-inline-figure` (run `npm i` in views)
7. open item in Authoring images, no errors, you can edit images
